sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeOperationResult.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Collection of <see cref="JobMetadata"/> objects corresponding
+    /// to a batch of documents, and information about the batch operation.
+    /// </summary>
+    public class AnalyzeOperationResult
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="entitiesResult">Entities result returned from the service</param>
+        /// <param name="piiEntitiesResult">PII Entities result returned from the service</param>
+        /// <param name="keyPhraseResult">Keyphrase result returned from the service</param>
+        /// <param name="map">Variable to map documents with id and maintaining the order</param>
+        /// <param name="displayName">DisplayName for the Job</param>
+        internal AnalyzeOperationResult(EntitiesResult entitiesResult, PiiEntitiesResult piiEntitiesResult, KeyPhraseResult keyPhraseResult, IDictionary<string, int> map, string displayName)
```

Comment:
could you rename map to something that will clarify what the purpose of the this variable?